brave/brave-core
# Show title for custom filter list subscriptions

Resolves https://github.com/brave/brave-browser/issues/28416

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Visit `brave://settings/shields/filters`
2. Add `https://filters.adtidy.org/extension/chromium/filters/17.txt` to the `Add custom filter lists` section
3. Verify that the title of the first column in the table that appears is `List title`
4. Verify that the entry under `List title` is `AdGuard URL Tracking filter`
5. Next, add `https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/TrackParamFilter/sections/general_url.txt` as second custom filter list
6. Verify that the title is just the URL of the list (truncated as necessary to fit on-screen)
7. Hover over each of the titles and ensure that the mouseover text shows the full original URLs used to subscribe to each.
